docs/guides/drivers/docker.md

```diff
@@ -0,0 +1,45 @@
+# Docker driver
+
+The buildx docker driver is the default builtin driver, that uses the BuildKit
+server components built directly into the docker engine.
+
+No setup should be required for the docker driver - it should already be
+configured for you:
+
+```console
+$ docker buildx ls
+NAME/NODE       DRIVER/ENDPOINT      STATUS   BUILDKIT PLATFORMS
+default         docker                                 
+  default       default              running  20.10.17 linux/amd64, linux/386
+```
+
+This builder is ready to build with out-of-the-box, requiring no extra setup,
+so you can get going with a `docker buildx build` as soon as you like.
+
+Depending on your personal setup, you may find multiple builders using the
+docker driver setup. For example, on a system that runs both the package
+managed version of dockerd, as well as Docker Desktop, you might have the
+following:
+
+```console
+NAME/NODE       DRIVER/ENDPOINT STATUS  BUILDKIT PLATFORMS
+default         docker                           
+  default       default         running 20.10.17 linux/amd64, linux/386
+desktop-linux * docker                           
+  desktop-linux desktop-linux   running 20.10.17 linux/amd64, linux/arm64, linux/riscv64, linux/ppc64le, linux/s390x, linux/386, linux/arm/v7, linux/arm/v6
+```
+
+Unlike the [other drivers](../index.md), builders using the docker driver
+cannot be manually created, and are automatically constructed from the
+available [Docker Contexts](https://docs.docker.com/engine/context/working-with-contexts/).
+Additionally, they cannot be configured to a specific BuildKit version, and
+cannot take any extra parameters.
+
```

Comment:
Add a sentence explaining that in order to add new Docker instances here user needs to create a `docker context` pointing to that instance, and then they will show up in the builders list.